hypothesisfuzzer/fuzzing/repo_fuzzer.py

```diff
@@ -19,15 +19,15 @@ class RepoFuzzer:
     def __init__(self, name, config):
 
         self.log = logging.getLogger('logger')
```

Comment:
again, make this global outside the class and with `__name__` instead of logger.